README.md

```diff
@@ -105,13 +106,20 @@ pip install vunit-hdl
 # Clone this repo and submodules
 git clone --recurse-submodules  https://github.com/phase4ground/dvb_fpga
 cd dvb_fpga
-# Generate test data
-cd gnuradio_data/
-make all
-cd ..
 # Run the tests
 ./run.py
 ```
+```sh
+cd dvb_fpga
+# list the test
+./run.py -l
+```
+```sh
+cd dvb_fpga
+# to run individual test
+./run.py -i
+```
+
```

Comment:
```suggestion
The first invocation of `run.py` will run GNURadio and create stimulus files.

To list tests use `./run.py -l`:
```sh
$ ./run.py -l
lib.axi_bit_interleaver_tb.data_width=8,all_parameters.back_to_back
lib.axi_bit_interleaver_tb.data_width=8,all_parameters.slow_master
lib.axi_bit_interleaver_tb.data_width=8,all_parameters.slow_slave
lib.axi_bit_interleaver_tb.data_width=8,all_parameters.both_slow
lib.axi_ldpc_table_tb.test_all_configs.back_to_back
lib.axi_ldpc_table_tb.test_all_configs.slow_master
lib.axi_ldpc_table_tb.test_all_configs.slow_slave
lib.axi_ldpc_table_tb.test_all_configs.slow_master,slow_slave
lib.axi_bch_encoder_tb.test_all_configs.back_to_back
lib.axi_bch_encoder_tb.test_all_configs.slow_master
lib.axi_bch_encoder_tb.test_all_configs.slow_slave
lib.axi_bch_encoder_tb.test_all_configs.both_slow
lib.dvbs2_tx_tb.test_all_configs.back_to_back
lib.axi_ldpc_encoder_core_tb.test_all_configs.back_to_back
lib.axi_ldpc_encoder_core_tb.test_all_configs.data=0.5,table=1.0,slave=1.0
lib.axi_ldpc_encoder_core_tb.test_all_configs.data=1.0,table=1.0,slave=0.5
lib.axi_ldpc_encoder_core_tb.test_all_configs.data=0.75,table=1.0,slave=0.75
lib.axi_ldpc_encoder_core_tb.test_all_configs.data=1.0,table=0.5,slave=1.0
lib.axi_ldpc_encoder_core_tb.test_all_configs.data=1.0,table=0.75,slave=0.75
lib.axi_ldpc_encoder_core_tb.test_all_configs.data=0.8,table=0.8,slave=0.8
lib.axi_baseband_scrambler_tb.test_all_configs.back_to_back
lib.axi_baseband_scrambler_tb.test_all_configs.slow_master
lib.axi_baseband_scrambler_tb.test_all_configs.slow_slave
lib.axi_baseband_scrambler_tb.test_all_configs.both_slow
Listed 24 tests
```

Tests above will run all parameter combinations, to run a specific test with a specific FECFRAME, code rate and modulation, use `./run.py -i <test_name>`. For example, `./run.py -i -l` will list all *individual* tests:

```sh
./run.py -i -l
lib.axi_bit_interleaver_tb.data_width=8,FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.back_to_back
lib.axi_bit_interleaver_tb.data_width=8,FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.slow_master
lib.axi_bit_interleaver_tb.data_width=8,FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.slow_slave
lib.axi_bit_interleaver_tb.data_width=8,FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.both_slow
...
lib.axi_ldpc_table_tb.FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.slow_slave
lib.axi_ldpc_table_tb.FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.slow_master,slow_slave
lib.axi_ldpc_table_tb.FrameType=short,ConstellationType=8PSK,CodeRate=1/4.back_to_back
lib.axi_ldpc_table_tb.FrameType=short,ConstellationType=8PSK,CodeRate=1/4.slow_master
lib.axi_ldpc_table_tb.FrameType=short,ConstellationType=8PSK,CodeRate=1/4.slow_slave
lib.axi_ldpc_table_tb.FrameType=short,ConstellationType=8PSK,CodeRate=1/4.slow_master,slow_slave
...
lib.axi_bch_encoder_tb.FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.back_to_back
lib.axi_bch_encoder_tb.FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.slow_master
lib.axi_bch_encoder_tb.FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.slow_slave
lib.axi_bch_encoder_tb.FrameType=normal,ConstellationType=8PSK,CodeRate=1/4.both_slow
lib.axi_bch_encoder_tb.FrameType=short,ConstellationType=8PSK,CodeRate=1/4.back_to_back
lib.axi_bch_encoder_tb.FrameType=short,ConstellationType=8PSK,CodeRate=1/4.slow_master
lib.axi_bch_encoder_tb.FrameType=short,ConstellationType=8PSK,CodeRate=1/4.slow_slave
lib.axi_bch_encoder_tb.FrameType=short,ConstellationType=8PSK,CodeRate=1/4.both_slow
...
Listed 592 tests
```

```

I think it's worth adding some context and examples of the commands output as well